### PR TITLE
Compile valgrind wrapper

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Relevant refs upstream:

- https://github.com/CodSpeedHQ/pytest-codspeed/issues/56
- https://github.com/CodSpeedHQ/pytest-codspeed/pull/57
- https://github.com/CodSpeedHQ/pytest-codspeed/releases/tag/v3.1.0

### CODE REVIEW

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
This is ready now @xhochy @kenodegard, thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,14 +12,18 @@ source:
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   noarch: python
```
Comment (jaimergp) [inline recipe/meta.yaml:14]:
```suggestion
```

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by xhochy** [comment]
Is this only visible with an import test? Couldn't we also execute something what did break in your job directly?

**Comment by jaimergp** [comment]
We'd need to create a pytest test file and run it. Not against it, but the import test looked simpler.

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by xhochy** [comment]
We probably need something here like we do for `aiohttp`: Build a noarch package for all platforms, but also have an arch-specific one for linux.

**Comment by jaimergp** [comment]
Upstream [only publishes linux wheels now](https://pypi.org/project/pytest-codspeed/#files), though. I'm not even sure this works outside Linux environments.

**Review by xhochy — APPROVED** [review]
Let's merge and see whether someone requests `osx` support.

**Comment by art049** [comment]
Hey just discovered this repo. 

As you mentionned, we just switched to prebuilt wheels in pytest-codspeed [v3.1.0]9https://github.com/CodSpeedHQ/pytest-codspeed/releases/tag/v3.1.0). 

> Upstream [only publishes linux wheels now](https://pypi.org/project/pytest-codspeed/#files), though. I'm not even sure this works outside Linux environments.

It works outside linux environment however the instrumentation capabilities are not enabled.

**Pull request merged**